tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -830,10 +830,14 @@ namespace Tensile
         bool     gsuc     = false; // initialized false
         bool     gsuwgmrr = false; // initialized false
         int32_t  wgm      = param.wgm() != 0 ? param.wgm() : sizeMapping.workGroupMapping;
+        uint32_t wgmxcc   = 0;
+        uint32_t wgmxccg  = 0;
         const uint32_t mask16       = 0xFFFF;
         const uint32_t mask14       = 0x3FFF;
+        const uint32_t mask12       = 0xFFF;
         const uint32_t mask8        = 0xFF;
         uint32_t       internalArg0 = 0;
+        uint32_t        internalArg1 = 0;
```

Comment:
not align, Have you ran the clang-format?

Reply:
updated.